lib/modules/dosomething/dosomething_kudos/dosomething_kudos.module

```diff
@@ -275,16 +275,16 @@ function dosomething_kudos_get_kudos_for_user_by_file($fid) {
 
 /**
  * Check to see if the kudos term has been selected by the user on specified
- * reportback.
+ * reportback item kudos data.
  *
- * @param  object $reportback_item
+ * @param  array $kudos
  * @param  string $term
  * @return bool
  */
-function dosomething_kudos_term_is_selected($reportback_item, $term) {
-  $reportback_item = (object) $reportback_item;
+function dosomething_kudos_term_is_selected($kudos, $term) {
+  $kudos = (object) $kudos;
```

Comment:
I had it from before when we were passing the RB object... but now it's just a kudos array; but not 100% sure if it's ever a Kudos object. Also, I just prefer arrow notation over array notation 😖